src/clauses/Return.test.ts

```diff
@@ -198,42 +209,42 @@ describe("CypherBuilder Return", () => {
 
             expect(queryResult.params).toMatchInlineSnapshot(`{}`);
         });
+    });
 
-        describe("Return distinct", () => {
-            test("Return distinct", () => {
-                const node = new Cypher.Node({
-                    labels: ["MyLabel"],
-                });
-                const returnQuery = new Cypher.Return(node, new Cypher.Literal(10)).distinct();
-
-                const queryResult = returnQuery.build();
-                expect(queryResult.cypher).toMatchInlineSnapshot(`"RETURN DISTINCT this0, 10"`);
-                expect(queryResult.params).toMatchInlineSnapshot(`{}`);
+    describe("Return distinct", () => {
```

Comment:
I think we can remove the extra `describe` for consistency with the other tests here if you agree

Reply:
There is another describe in this same file, I'm not sure there is a clear cut have describes or not :sweat_smile: